content/en/threat-model.md

```diff
@@ -0,0 +1,123 @@
+|                        |                  |               | x                     |              | x             |              |             | Same as Fulcio monitor + Rekor monitor                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+|                        |                  | x             |                       |              |               |              |             | Can remove certificates (invalidate them) by indicating that they are not in the log when Bob queries, fork attack by showing different times or certificates to different users (for things Fulcio has signed), and may block the issuance of Fulcio certificates by not creating an SCT.  Deletion / fork may be detected by Fulcio CT log monitors.                                                                                                                                                                                                   |
+|                        |                  |               |                       | x            |               |              |             | Can change the timestamp on Fulcio signatures used outside the validity window to make them appear valid when Bob queries.  Can deny the existence of valid signatures leading to them being untrusted.  Can later recover using offline delegation in Sigstore’s TUF root of trust.  Deletion or fork attacks may be detected by Rekor monitors. Changes to the Rekor CT log would be challenging to recover as it eliminates the append-only promise of Rekor.                                                                                         |
+|                        |                  | x             | x                     |              |               |              |             | Same as Fulcio CT log, but bad behavior will not be detected if all monitors are compromised, so replay and fork attacks are undetected.                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
This is the first mention of "replay attacks".  Previously, only deletion and fork attacks were defined.

I wonder if this makes more sense as a matrix of attacks + infrastructure required to compromise, e.g.

| capability | Fulcio CA server | Fulcio CT log | Fulcio monitor | Rekor server | Rekor CT log | Rekor monitor | Sigstore Root of Trust | OIDC account | OIDC server |
|---|---|---|---|---|---|---|---|---|---|
| Invalidate a signed cert by XYZ |  |  |  |  | X | D | | | |
| Add a signature by re-using a cert |  | X | D | | | | |

etc

Where `X` indicates the attack is possibly with those components compromised, and `D` indicates that detection can additionally be defeated with the addition components compromised.

Of course, even making that small table kinda sucked, so I can understand not wanting to re-pivot everything.  😬

Reply:
I made some other clarifications to the table, but skipped this one for now. In a future iteration we can re-visit the structure